locale/en/blog/release/v6.0.0.md

```diff
@@ -0,0 +1,172 @@
+---
+date: 2016-04-26T20:35:53.135Z
+version: 6.0.0
+category: release
+title: Node v6.0.0 (Current)
+slug: node-v6-0-0
+layout: blog-post.hbs
+author: James M Snell
+---
+
+It's here! Just three days shy of exactly six months after we released Node v5.0.0, the Node.js developers are happy to announce that Node.js v6 is available and ready for use! 
+
+This new major version of Node.js delivers significant improvements in performance, reliability, usability and security. Module loading is nearly four times faster than in Node.js v4, tests and documentation have seen significant improvements, usability of the Buffer and File System APIs has been improved, and Node.js now supports 93% of JavaScript language features thanks to V8 version 5.0.
+
+In accordance with our [Long Term Support plan](https://github.com/nodejs/LTS/), Node.js v4 'Argon' will continue to be supported as the *Active* Long Term Support stream until April of 2017, after which it will transition into what we called *Maintenance* LTS. The new Node.js v6 branch will become the new *Active* Long Term Support stream in October of this year, and will continue to be supported actively until April of 2018. If our LTS plan is new to you, [read more](https://medium.com/@nodesource/essential-steps-long-term-support-for-node-js-8ecf7514dbd#.z6bzdfogi) about how it works so that you can make a properly informed decision regarding which version of Node.js to adopt.
+
+The general rule for deciding which version of Node.js to use is:
+
+* Stay on or upgrade to Node.js v4 (currently at v4.4.3) if you need stability and have a complex production environment.
+* Upgrade to Node.js v6 if you have the ability to upgrade versions quickly and want to play with the latest features as they arrive.
+
+The release notes below are annotated with the main breaking changes that warrant the jump to v6. Note that because this new version of Node.js is shipping with a new version of V8, you will need to recompile any native addons you have already installed or you will receive a runtime error when trying to load them. Use `npm rebuild` or simply remove your *node_modules* directory and `npm install` from scratch.
+
+### What about Node.js v5?
+
+Node.js v5 will continue to be supported for the next two months in order to give developers currently using v5 time to transition to Node.js v6.
+
+### What about Node.js v0.10 and v0.12?
+
+If you're still currently using Node.js v0.10 or v0.12, it is time to begin the transition to v4 or v6. Both of these versions are considered to be in *Maintenance* mode currently and will fall off our support plan completely later this year.
```

Comment:
> Both of these versions
you may want to be explicit that you are referring to v0.10 and v0.12 here to avoid confusing as you mentioned two pairs of versions